cabal-install/src/Distribution/Client/Setup.hs

```diff
@@ -2472,3 +2495,17 @@ relevantConfigValuesText :: [String] -> String
 relevantConfigValuesText vs =
      "Relevant global configuration keys:\n"
   ++ concat ["  " ++ v ++ "\n" |v <- vs]
+
+
+-- ------------------------------------------------------------
+-- * Commands to support show-build-info
+-- ------------------------------------------------------------
+
+showBuildInfoCommand :: CommandUI ShowBuildInfoFlags
```

Comment:
This seems to be support for `v1-` commands, but I don't think it is actually wired in, so, Ill just remove that.